sbi/src/error.rs

```diff
@@ -24,6 +24,10 @@ pub enum Error {
     AlreadyStarted = -7,
     /// Some of the given counters have already been stopped.
     AlreadyStopped = -8,
+    /// Some resources are busy and could not process the request.
+    Busy = -9,
+    /// The response to the SBI call is too large and overflows the allocated capacities.
+    Overflow = -10,
```

Comment:
Have these been added to the spec?

Reply:
We're proposing to add those with the attestation SBI extension:
https://github.com/sameo/riscv-sbi-doc/pull/1/files

Reply:
A related question about the semantics (and the associated name). Since the response would overflow the passed in buffer, the presumably the passed-in input buffer was too small, so perhaps it should be InsufficientCapacity / BufferTooSmall, or something like that?

Reply:
The passed in buffer could be valid (The attestation caps structure has a known max size), but the SBI implementation could try to write passed the maximum number of allowed  registers. I think we should look a this as a different error than passing too small of a buffer.

Reply:
So this is really an internal error and there's nothing the caller can do about it, right? If that's the case, would `Failed` be more appropriate?

Reply:
That's less intrusive, yes. I'll introduce the 2 new errors with the PRs introducing the measurement registers read and extension implementations.